client/my-sites/themes/themes-banner/index.jsx

```diff
@@ -0,0 +1,53 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cssSafeUrl from 'lib/css-safe-url';
+
+export class ThemesBanner extends Component {
+	static propTypes = {
+		title: PropTypes.string.isRequired,
+		description: PropTypes.string.isRequired,
+		ctaLabel: PropTypes.string.isRequired,
+		ctaAction: PropTypes.func,
+		backgroundImage: PropTypes.string,
+	};
+
+	renderButton = ( label, action, primary ) => {
```

Comment:
We try to avoid thin `render...()` wrapper methods like this and just inline the relevant component inside the main `render()` method.